lib/realtime/api.ex

```diff
@@ -77,6 +77,8 @@ defmodule Realtime.Api do
 
   """
   def update_tenant(%Tenant{} = tenant, attrs) do
+    drop_dist_cache(tenant.external_id)
```

Comment:
@abc3 did you confirm with Infra that this is alright with timeouts on their end? This adds 10 seconds to the database operation after this for a total of 25 seconds max.

Also, for this to take a max of 25 seconds seems way too long for this method call.

Reply:
@abc3 we should call `drop_dist_cache/1` in `create_tenant/1` as well to cover the edge case.

Reply:
what edge cases do you mean?

Reply:
How you calculated 25 seconds? 10 seconds is the maximum timeout to go over all cluster, infra doesn't need to know about this timeout anything

Reply:
`10 seconds is the maximum timeout` + 15 seconds for database update max timeout.

Well there was an issue previously where we added something that extended the max timeout of a method call and that affected Infra.

Reply:
There was a blocking operation to inform tenant's users to stop. Currently, 15 seconds should be enough for everything.